client/setup.py

```diff
@@ -71,6 +71,7 @@ def _format_version() -> str:
     url="https://github.com/star-whale/starwhale",
     license="Apache License 2.0",
     packages=find_packages(exclude=["ez_setup", "tests*"]),
+    package_data={"": ["template/*"]},
```

Comment:
`template/*` --> `"template/*.tmpl"` or `core/runtime/template/*.tmpl` ?
use the more accurate patten?

Reply:
fixed